llvm/lib/Target/SPIRV/SPIRVTargetTransformInfo.h

```diff
@@ -37,6 +38,15 @@ class SPIRVTTIImpl : public BasicTTIImplBase<SPIRVTTIImpl> {
   explicit SPIRVTTIImpl(const SPIRVTargetMachine *TM, const Function &F)
       : BaseT(TM, F.getDataLayout()), ST(TM->getSubtargetImpl(F)),
         TLI(ST->getTargetLowering()) {}
+
+  TTI::PopcntSupportKind getPopcntSupport(unsigned TyWidth) {
```

Comment:
TTI probably should have a better default checking if the operation is legal. It's also not great that TargetLowering also has isCtpopFast

Reply:
Done.